src/obfl-specification.html

```diff
@@ -1318,7 +1320,7 @@ <h4 id="L849" class="include">span</h4>
 <div class="sidebar markup">
 <dl>
   <dt>Usage</dt>
-    <dd>before, after, toc-entry, block, item</dd>
+    <dd>before, after, toc-entry, block, toc-block, item</dd>
```

Comment:
As ```toc-block``` is used for nesting and text is not allowed then ```span``` is not required or needed.